### PR TITLE
chore: Updates from coach review

### PR DESCRIPTION
**Pull request by edine-noella**
- updated the bg-color for page
- reduced height of the reference section
- increased text-size of the about us secton

### CODE REVIEW

File: src/sections/about.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<div class="mx-auto w-12/12 lg:w-8/12 mt-20 sm:mt-40 sm:w-11/12 ">
+<div class="mx-auto w-12/12 lg:w-8/12 mt-20 sm:mt-56 sm:w-11/12 ">
```
Comment (AL2002MI08) [inline src/sections/about.html:1]: Remove space at the end of the div.

File: src/sections/footer.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <footer class="bottom-0 pb-9">
   <div class="py-24 hidden sm:block w-6/12 sm:6/12 md:12/12 mx-auto">
-    <img src="src/assets/images/red-vector.svg" alt="" class="m-auto" />
```
Comment (tuyishimejohnson) [inline src/sections/footer.html:5]: Can you add the description inside the alt attribute?

File: src/sections/footer.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
+    <a href="../../data-privacy.html"><img src="src/assets/images/red-vector.svg" alt="" class="m-auto" /></a>
```
Comment (tuyishimejohnson) [inline src/sections/footer.html:5]: Even here.

Comment (AL2002MI08) [inline src/sections/footer.html:5]: Remove space at the end of the div

**Review by tuyishimejohnson — CHANGES_REQUESTED** [review]

**Review by AL2002MI08 — CHANGES_REQUESTED** [review]
Remove trailing spaces at the end of the div

**Review by AL2002MI08 — APPROVED** [review]
Great work done!

**Pull request merged**